flink-state-backends/flink-statebackend-rocksdb/src/main/java/org/apache/flink/contrib/streaming/state/RocksDBOptions.java

```diff
@@ -40,7 +43,14 @@
                     .noDefaultValue()
                     .withDeprecatedKeys("state.backend.rocksdb.checkpointdir")
                     .withDescription(
-                            "The local directory (on the TaskManager) where RocksDB puts its files.");
+                            Description.builder()
+                                    .text(
+                                            "The local directory (on the TaskManager) where RocksDB puts its files. Per default, it will be <WORKING_DIR>/tmp. See %s for more details.",
```

Comment:
if `<workingDir>/tmp` is always cleaned when the process starts, then what do we gain by putting rocksdb files in there? I don't quite understand the justification in the FLIP.

Reply:
What we gain is that the working directory of RocksDB and the local state store directory are by default located on the same volume. This allows us to use hard links for copying the sst files. If they were located on different volumes, then we would have to copy the files over which is obviously slower. So long story short, it gives us a better default behaviour.